index.js

```diff
@@ -1,33 +1,36 @@
 // Promiseの基本パターン
-function countdown (seconds) {
-  return new Promise((resolve, reject) => {
-    const timeoutIds = []
-    for (let i = seconds; i >= 0; i--) {
+function countdown(seconds) {
+  return new Promise((onFulfilled, onRejected) => {
+    const timeoutIds = [];
+    for(let i=seconds; i >=0; i--) {
       timeoutIds.push(setTimeout(
         () => {
-          if (i === 13) {
-            timeoutIds.forEach(clearTimeout) /* すべてクリア */
-            reject(new Error(`${i}という数は不吉過ぎます`))
-          } else if (i > 0) {
-            console.log(i + `...`)
-          } else {
-            console.log('GO!')
-            resolve()
+          if(i===13) {
+            timeoutIds.forEach(clearTimeout); /* すべてクリア */
+            onRejected(new Error(`${i}という数は不吉過ぎます`));
+          }
+          else if(i>0) {
+            console.log(i + `...`);
+          }
+          else {
+            console.log("GO!");
+            onFulfilled();
           }
         }, /* 無名関数の終わり */
-        (seconds - i) * 1000))
-    }
-  })
+        (seconds-i)*1000)
+      )}
```

Comment:
ERROR: Closing curly brace should be on the same line as opening curly brace or on the line after the previous block. (brace-style)